src/main/resources/application.yaml

```diff
@@ -24,3 +24,6 @@ ftp:
   fingerprint: ${FTP_FINGERPRINT}
   user: ${FTP_USER}
   password: ${FTP_PASSWORD}
+  downtime:
+    from: ${FTP_DOWNTIME_FROM:17:00}
```

Comment:
Maybe there shouldn't be any defaults here?

Reply:
this is the actual time for Jira story